src/execution_plan/ops/op_create.c

```
@@ -193,25 +204,39 @@ static Record CreateConsume
 	return _handoff(op);
 }
 
-static OpBase *CreateClone(const ExecutionPlan *plan, const OpBase *opBase) {
+static OpBase *CreateClone
+(
+	const ExecutionPlan *plan,
+	const OpBase *opBase
+) {
 	ASSERT(opBase->type == OPType_CREATE);
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)